manifests/05_deploy-ibm-cloud-managed.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+  labels:
+    app: service-ca-operator
+  name: service-ca-operator
```

Comment:
what is this file? Why is it called ...-ibm-cloud-managed...

Reply:
It's a generated file (from 05_deploy.yaml) specific to the `ibm-cloud-managed` profile. The file is created by applying the patch in `profile-patches/ibm-cloud-managed/05_deploy.yaml-patch`. 
`make verify` ensures that the file is up to date (by applying the patch)
`make update` updates the file when either the patch or the source manifest changes.

Reply:
The reason for this, is that for any CVO-delivered deployment that will be applied in a ROKS cluster (or for that matter any hosted control plane deployment of openshift), we need to remove the node selector for masters, since in that environment there are no master machines.

Reply:
I see. So when we change the main file without the patch, `make verify` will complain.

Reply:
Yes